src/components/firstProfile/uploadPhotos/UploadPhotos.tsx

```diff
@@ -100,6 +100,10 @@ const useStyles = makeStyles()(() => ({
     marginLeft: 20,
     marginRight: 20,
     paddingTop: 50,
+    '@media (max-width: 600px)': {
+      maxWidth: '380px',
+      width: '95vw',
+    },
```

Comment:
Why we need this `'@media (max-width: 600px)`?
As for me, it looks good without this style. Maybe I missed something